package.json

```diff
@@ -46,6 +46,7 @@
     "node-sass": "4.12.0",
     "react": "16.8.6",
     "react-dom": "16.8.6",
+    "react-router-dom": "^5.0.1",
```

Comment:
You'll want to pin this (you can run yarn add with `--exact` in the future for convenience), or renovate will be cross with you.